### PR TITLE
Added a limit to the number of characters in PayMovie

### PR DESCRIPTION
**Pull request by SketchMaster2001**
I also changed an integer that should be boolean.

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -141,9 +141,9 @@ class PayMovies(db.Model):
     aspect = db.Column(db.Boolean, nullable=False)
     payenddt = db.Column(db.String(19), nullable=False)
     ds_dist = db.Column(db.Boolean, nullable=False)
-    ds_mov_id = db.Column(db.Integer)
+    ds_mov_id = db.Column(db.Boolean, nullable=False)
```
Comment (spotlightishere) [inline models.py:144]: This should be an integer - `dsdist` is a boolean specifying its existence, where `dsmovid` specifies what movie ID it should download to package and send to the DS. We may want to better document how each field is used somewhere in source as well, alongside the Wii no Ma docs repository.

**Review by spotlightishere — CHANGES_REQUESTED** [review]
Appreciate the change!

File: models.py (hunk 1 above)
Comment (SketchMaster2001) [reply models.py:144]: ah I'll fix it

Comment (SketchMaster2001) [reply models.py]: Fixed

**Review by spotlightishere — APPROVED** [review]

**Pull request merged**